ui/apps/platform/src/Containers/Vulnerabilities/NodeCves/Node/useNodeVulnerabilities.ts

```diff
@@ -26,7 +26,7 @@ export default function useNodeVulnerabilities({
 }: { nodeId: string; query: string } & ClientPagination) {
     return useQuery<
         {
-            node: {
+            node?: {
```

Comment:
An unrelated issue I found when I accidentally had an API response mocked in local dev. This type can techincally return `null`, which if not accounted for will cause page crashes.